packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -80,29 +80,40 @@ export class LitDevTutorial extends LitElement {
   private _idx = 0;
 
   /**
-   * Message to be displayed if there was an error loading
+   * Preloaded HTML content for the next step (idx is explicit to ensure we
+   * don't get out of sync).
    */
   @state()
-  private _htmlLoadError = '';
+  private _preloadedHtml?: {
+    idx: number;
+    promise: Promise<{html: string; isOk: boolean}>;
+  };
 
   /**
-   * Message to be displayed if there was an error loading
+   * Tutorial manifest json file
    */
-  @state()
-  private _manifestLoadError = '';
+  private get _manifest(): TutorialManifest {
+    return this._manifestTask.value?.manifest ?? {steps: [{title: ''}]};
+  }
 
   /**
-   * Preloaded HTML content for the next step (idx is explicit to ensure we
-   * don't get out of sync).
+   * Message to be displayed if there was an error loading
    */
-  @state()
-  private _preloadedHtml?: {idx: number; promise: Promise<string>};
+  private get _manifestLoadError() {
+    return this._manifestTask.value?.isOk === false
+      ? `Could not fetch tutorial manifest. Invalid URL /${this._projectLocation}/`
+      : '';
+  }
 
   /**
-   * Tutorial manifest json file
+   * Message to be displayed if there was an error loading
    */
-  private get _manifest(): TutorialManifest {
-    return this._manifestTask.value ?? {steps: [{title: ''}]};
+  private get _htmlLoadError() {
+    return this._htmlTask.value?.isOk === false
+      ? `Could not fetch step description. Invalid URL /${
```

Comment:
Nit: If a someone encounters this error, it may be useful to include a call to action, since it appears in the tutorial text area. Maybe a link to the tutorials home page? Non blocking.

For example, `https://lit.dev/tutorials/intro-to-lit/#9` is an invalid link that shows code but not step description.
Alternatively I'm uncertain about putting the error message in the content area and having it look similar to content.

Reply:
It actually just shows up as `/intro-to-lit/#09` I didn't want to put an actual working link here. Besides I'm pretty sure we'd be more likely to run into this as authors rather than users. 

Probably fits better in a toast or a dialog, but I feel like that's a bunch of effort for MVP. Will write a TODO

Reply:
Added